### PR TITLE
Update to latest sentry version

### PR DESCRIPTION
**Pull request by k-nut**

### CODE REVIEW

**Review by ticosax — APPROVED** [review]

**Comment by k-nut** [comment]
@ticosax I updated this to also use poetry now. Would be great if you could give it another quick look.

File: pyproject.toml [hunk 1]
```diff
@@ -24,13 +24,13 @@ Flask-Cors = "3.0.9"
...
 Flask-Migrate = "2.1.1"
 Flask-SQLAlchemy = "2.5.1"
-raven = {extras = ["flask"], version = "^6.10.0"}
 psycopg2 = "2.9.3"
 simplejson = "3.12.0"
 fints = "3.0.1"
 gevent = "20.12.1"
 gunicorn = "20.0.4"
 MarkupSafe = "2.0.1"
+sentry-sdk = {extras = ["flask"], version = "^1.5.5"}
```
Comment (ticosax) [inline pyproject.toml:33]: Then, we let dependabot make a pull request for every new upgrades. If it's green we merge, if not we change our code for any deprecated usage of the lib.
```suggestion
sentry-sdk = {extras = ["flask"], version = "*"}
```

File: solawi/app.py [hunk 2]
```diff
@@ -25,6 +26,10 @@
     # which is why we re-write it to match what SQLAlchemy expects.
     db_url = db_url.replace("postgres://", "postgresql://")
 
+sentry_sdk.init(
+    dsn=os.environ.get("SENTRY_DSN"), integrations=[FlaskIntegration()], traces_sample_rate=0.6
```
Comment (ticosax) [inline solawi/app.py:30]: There is also `sqlalchemy` and  `requests` integration we could add.

**Review by ticosax — APPROVED** [review]

**Comment by k-nut** [comment]
Ah, is it convention to just put `*` version into the `pyproject.toml` file itself and then let dependabot update the lockfile? I'm only used to the npm ecosystem lately where one would pick a specific version in the `package.json` as well and then would have dependabot update both the `lockfile` and the `package.json` if applicable.

**Comment by ticosax** [comment]
> Ah, is it convention to just put `*` version into the `pyproject.toml` file itself and then let dependabot update the lockfile? I'm only used to the npm ecosystem lately where one would pick a specific version in the `package.json` as well and then would have dependabot update both the `lockfile` and the `package.json` if applicable.

It's a deliberate decision we opt in 
https://github.com/k-nut/csa-share-management/blob/master/.github/dependabot.yml#L7
It doesn't change the fact that the lock file remain source of truth (predictable builds), it simplify the management of the versions because the hash of the `pyproject.toml` file doesn't change at every new upgrade. And mostly, the locking per major versions ("^1.3") is arbitrary and is preventing us more from getting perfectly usable upgrades rather than defending us from major bumps that breaks internal API.
By experience, it's the best available option here.

**Comment by k-nut** [comment]
Ah I see, thanks for the explanation!

**Pull request merged**